spec/models/rook_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'rails_helper'
+
+RSpec.describe Rook, type: :model do
+  before do
+    # Note: We want the board to be emptry for this test.
+    allow_any_instance_of(Game).to receive(:populate_board!).and_return true
+  end
+
+  describe '#valid_move?' do
+    before :each do
+      @game = FactoryGirl.create(:game)
+      @white_rook = FactoryGirl.create(:white_rook, game: @game)
+      @black_rook = FactoryGirl.create(:black_rook, game: @game)
+    end
```

Comment:
you can also do this with `let!`. Which is a bit more concise.

```Ruby
let!(:game) {  FactoryGirl.create(:game) }
let!(:white_rook) {  FactoryGirl.create(:white_rook, game: game) }
let!(:black_rook) {  FactoryGirl.create(:black_rook, game: game) }
```

Then you don't need the instance variables. and can just call `game`... `white_rook` etc